src/chart/line/LineView.ts

```diff
@@ -1028,7 +1030,8 @@ class LineView extends ChartView {
 
         polyline = new ECPolyline({
             shape: {
-                points
+                points,
+                loop
```

Comment:
Can `ECPolygon` be used if `loop` is true? If so, you don't have to implement the loop logic.

Reply:
Yes it can, what do you mean I don't have to implement it?

Reply:
Yes. I think if it passes the test, you can use `ECPolygon` and remove [getPoints](https://github.com/apache/echarts/pull/17720/files#diff-d8a9111d50b6c82ff1bceceef47695d31bd3b1d4cd219d6938bc654cae5056c1R214).